### ISSUE TITLE
curdir not inserted into sys.path in trial, twistd etc. on Windows

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10409]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#554|
|Type|defect|
|Created|2004-03-17 21:06:15Z|

<details><summary>Searchable metadata</summary>

```
trac-id__554 554
type__defect defect
reporter__itamarst itamarst
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1079557575000000 1079557575000000
changetime__1475837360252857 1475837360252857
version__ 
owner__ 
cc__itamarst
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
if hasattr(os, "getuid") and os.getuid() != 0:
    sys.path.insert(0, os.curdir)

means that the insert never happens on Windows, though it should.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
Fixed in r10742.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc commented|
|-|-|

In changeset 93c23565953b6938a90bc34cc5a5b646fd8acbbf
```
#!CommitTicketReference repository="" revision="93c23565953b6938a90bc34cc5a5b646fd8acbbf"
Merge pull request #554 from twisted/8851-rodrigc-conch

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8851

Enable twisted.conch.test.test_helper on Python 3
```
